lib/command/run.js

```diff
@@ -18,17 +19,156 @@ module.exports = function (suite, test, options) {
   if (options.override) {
     config = deepMerge(config, JSON.parse(options.override));
   }
-  try {
-    codecept = new Codecept(config, options);
-    codecept.init(testRoot, function (err) {
-      if (err) throw new Error('Error while running bootstrap file :' + err);
-      codecept.loadTests();
-      codecept.run(test);
-    });
-  } catch (err) {
-    output.print('');
-    output.error(err.message);
-    output.print('');
-    output.print(output.colors.grey(err.stack.replace(err.message, '')));
+
+  let outputDir = config.output;
+  let defaultWindowSize = searchValueInObj('windowSize', config.helpers)
+
+  // run multiple codeceptjs with options
+  if (options.multiple) {
+
+    // get list of multiple options
+    let multipleOptions = options.multiple.replace(/ /g, "").split(';');
+    let browserList = ['chrome', 'firefox', 'ie', 'safari', 'edge'];
+
+    //iterate all multipleOptions
+    for (let i = 0; i < multipleOptions.length; i++) {
+
+      // get suites
+      let browserSuites = multipleOptions[i].split(':');
+
+      // if key in browser list, then seach suites for this browser
+      if (browserList.indexOf(browserSuites[0]) > -1) {
+        let finalSuites = browserSuites[1].split(',')
+
+        // iterate suites for browser
+        for (let j = 0; j < finalSuites.length; j++) {
+
+          runnerHelper(browserSuites[0], finalSuites[j], config)
+
+        }
+      } else if (browserSuites[0] == 'all' && browserSuites.length == 1) {
+
+        //get all suites for all browsers and run them all
+        for (let l = 0; l < Object.keys(config.multiple).length; l++) {
+          let browser = Object.keys(config.multiple)[l];
+          let configuration = config.multiple[browser];
+          if (Object.keys(configuration).length > 0) {
+            for (let j = 0; j < Object.keys(configuration).length; j++) {
+              let suite = Object.keys(configuration)[j]
+
+              runnerHelper(browser, suite, config)
+
+            }
+          }
+
+        }
+      } else if (browserSuites.length == 1 && browserSuites[0] != '') {
+
+        //run suite in all browsers
+        let suites = browserSuites[0].split(',');
+
+
+        for (let j = 0; j < suites.length; j++) {
+          for (let l = 0; l < Object.keys(config.multiple).length; l++) {
```

Comment:
not sure, but can this be written using forEach or `for..in` which looks nicer to me for languages other than C :)